contracts/contracts/mocks/uptime_feed_mock.cairo

```diff
@@ -0,0 +1,55 @@
+%lang starknet
```

Comment:
Why was this mock contract added? What is it used for?

The only usage I see is in a test that checks the interface of a mocked contract, why?

Reply:
To ensure work of interfaces and propagation of values through `libraries`. ie if deployed `SimpleReadAccessController` and calling has access then `SimpleReadAccessController::has_access` will be called, Same with `SimpleWriteAccessController`

Reply:
Remove this mock or leave it?